index.md

```diff
@@ -2,25 +2,29 @@
 
-The DevSecOps Maturity Model, which is presented in the talk, shows security measures which are applied when using DevOps strategies and how these can be prioritized.
-With the help of DevOps strategies security can also be enhanced. For example, each component such as application libraries and operating system libraries in docker images can be tested for known vulnerabilities.
-Attackers are intelligent and creative, equipped with new technologies and purpose. Under the guidance of the forward-looking DevSecOps Maturity Model, appropriate principles and measures are at hand implemented which counteract the attacks.
+With the advent of Agile frameworks, Enterprises and SMBs are shipping software - fast! According to the [Verizon 2020 Breach Investigation Report](https://enterprise.verizon.com/resources/reports/2020-data-breach-investigations-report.pdf) and [anecdotal evidence](https://www.equifaxbreachsettlement.com/) we see from the macro environment, application vulnerabilities are increasingly the most common attack vector. There are a lot of reasons why we as developer first organizations shelve security as a backlog item for some future sprint. A few reasons could be: security is hard, we have feature deadlines to meet, security is disruptive to my dev workflows. **The DevSecOps Maturity Model is an opinionated developer first framework for evaluating application security practices to ensure the applications we are shipping and their destination(s) are secure.** DSOMM seeks to embody the principles of "[DevSecOps](https://resources.github.com/whitepapers/Architects-guide-to-DevOps/)" by enabling developers and security engineers to partner in a _joint ownership model_ of the applications they are delivering.   
+
+Attackers are intelligent and creative, equipped with new technologies and purpose. Under the guidance of the forward-looking DevSecOps Maturity Model, the right culture, people, processes and tools are in place to effectively reduce the attack surface of your applications and therefore thwarting bad actors.
 
 ## Implementation Level
 ![Implementation Level](/assets/images/impl.png)
 
-# More Information: [dsomm.timo-pagel.de](https://dsomm.timo-pagel.de)
+# More Information: 
+* [dsomm.timo-pagel.de](https://dsomm.timo-pagel.de)
+* [GitHub Blog](https://github.blog/2020-08-06-achieving-devsecops-maturity-with-a-developer-first-community-driven-approach/)
+* [White Paper](https://resources.github.com/whitepapers/achieving-devsecops-maturity-github/)
+* [Education](https://courses.practical-devsecops.com/p/devsecops-professional)
 
 ## Project Leader
 <a href="mailto://timo.pagel@owasp.org">Timo Pagel</a>
+<a href="mailto://alwell-kevin@github.com">Kevin Alwell</a>
```

Comment:
Please start with improvements of the model. After a while you become "Collaborator". After we build a trust relationship you might become leader.
See also https://owasp.org/www-pdf-archive/PROJECT_LEADER-HANDBOOK_2014.pdf

Reply:
Sounds good, I thought this repo was mostly unsupported until your fast response :)

Reply:
This repo is a "placeholder". The real works lies in https://github.com/wurstbrot/DevSecOps-MaturityModel

Reply:
Oh, interesting! I made the assumption since it was under the OWASP org that this was the official work. Thanks for pointing that out.